srcpkgs/font-latinmodern/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'font-latinmodern'
+pkgname=font-latinmodern
+version=2.004
+revision=1
+depends="font-util"
+short_desc="Improved version of Computer Modern fonts as used in LaTeX"
+maintainer="classabbyamp <void@placeviolette.net>"
+license="custom:GFL"
```

Comment:
I think it's `LPPL-1.3c` is enough. The "requested but not required" means nothing needs to be done.

Reply:
I think GFL still needs to be mentioned, because that clause is still there even if what is does is optional